src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/templates/.content.xml

```diff
@@ -7,6 +7,9 @@
 #if ( $isSpaProject )
     <spa-app-template />
     <spa-page-template />
+#end
+#if ( $includeFormsheadless )
+    <blank-af-v2/>
```

Comment:
blank-af-v2 is required for forms core components. it should be included irrespective of headless flag.